### PR TITLE
 add wrappers to the methods to list moderators of channels and groups. 

### PR DESCRIPTION
**Pull request by nemhods**
Relevant API docs:

* https://rocket.chat/docs/developer-guides/rest-api/groups/moderators/
* https://rocket.chat/docs/developer-guides/rest-api/channels/moderators/

### CODE REVIEW

**Comment by jadolg** [comment]
Hi, @nemhods thanks for your contribution :)
Can you check your code in order to correct the issues why the build failing?
https://travis-ci.org/jadolg/rocketchat_API/jobs/517323868

File: tests/tests.py [hunk 1]
```diff
@@ -350,10 +350,8 @@ def test_channels_list_moderator(self):
         me = self.rocket.me().json()
         channels_add_moderator = self.rocket.channels_add_moderator(
```
Comment (jadolg) [inline tests/tests.py]: Usually testing if the API call returned success is a good idea. This way this check `local variable 'channels_add_moderator' is assigned to but never used` will be gone.

File: tests/tests.py [hunk 2]
```diff
@@ -350,10 +350,8 @@ def test_channels_list_moderator(self):
...
             'GENERAL', me.get('_id')).json()
-        print(channels_add_moderator)
         channels_list_moderator = self.rocket.channels_moderators(
             room_id='GENERAL').json()
-        print(channels_list_moderator)
         self.assertTrue(channels_list_moderator.get('success'))
         channels_remove_moderator = self.rocket.channels_remove_moderator(
```
Comment (jadolg) [inline tests/tests.py]: Same here. If you don't need these variables just remove them, but I would advise to just checking if the API call succeeded.

File: tests/tests.py (hunk 1 above)
Comment (nemhods) [reply tests/tests.py]: Sorry, didn't see that comment. I think I should leave out adding/removing moderators before testing the list moderators API call. It should succeed even without any moderators in place, and my setup code just convolutes the test.
Give me a second...

**Comment by jadolg** [comment]
great!

**Comment by jadolg** [comment]
great!
next step is trying to match the existing tests coverage.
take a look at [this](https://codecov.io/gh/jadolg/rocketchat_API/compare/12280dee62dbb745864d18c7d3a3926a2a63e358...b93bad7a9df7df67c9faa57c888e38f3be088bbe/diff) diff. it shows the lines that are never visited by the new tests.

**Comment by nemhods** [comment]
Thanks for walking me through this @jadolg 
Hopefully that last commit is up to the task.

**Comment by jadolg** [comment]
Everything looks green now :)
Thanks a lot for being patient and for contributing with your time and effort to this project :)
I'll merge your changes and make a new release ASAP

**Comment by jadolg** [comment]
Your code is already live in version 0.6.29 :)

**Pull request merged**